src/main.js

```diff
@@ -55,6 +59,26 @@ const filterBeforeCreate = (toast, toasts) => {
 
 const app = createApp(App).use(store).use(router);
 
+if (
+  ["staging", "production"].includes(process.env.NODE_ENV) &&
+  process.env.VUE_APP_SENTRY_DSN
+) {
+  // Since Vue3 isn't officially supported yet by Sentry, we're using
+  // the JavaScript Sentry integration with @sentry/browser.
+  // Refer Sentry documentation for more configs: https://docs.sentry.io/platforms/javascript/configuration/
+  Sentry.init({
+    dsn: process.env.VUE_APP_SENTRY_DSN,
+    integrations: [new Integrations.BrowserTracing()],
+    // Set tracesSampleRate to 1.0 to capture 100%
+    // of transactions for performance monitoring.
+    tracesSampleRate: 1.0,
+    environment: process.env.NODE_ENV,
+  });
+  app.config.errorHandler = (err) => {
+    Sentry.captureException(err);
```

Comment:
we also need to set this up for analytics too right?

Reply:
I tried looking this for CubeJS but couldn't find a way yet. I have asked this on their forum as well and awaiting reply.

As it's mostly API calls, I believe we will still get to know from Frontend Sentry if an analytics API endpoint fails. However, to capture more info, we will need something at CubeJS end as well.
I'm keeping a track here: [Analytics issue](https://github.com/avantifellows/plio-analytics/issues/22)

Reply:
Cool. We can look at it later if we need to.